docs/UserGuide.md

```diff
@@ -3,271 +3,478 @@ layout: page
+* `list` followed by `delete 2` deletes the 2nd person in the address book.
+
+  ![delete 2](images/delete-2.png)
+
+* `find n/Charlotte` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+
+  ![delete charlotte](images/delete-charlotte.png)
+
+[Return to Table of Contents](#table-of-contents)
+
+
+### Sort list of clients: `sort`
+
+**Purpose**: Sorts the current list of clients in ClientBook.
+
+**Format**: `sort -ATTRIBUTE -DIRECTION`
 
 * Sorts the list of clients according to the specified `ATTRIBUTE` and `DIRECTION`.
-* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy.
-* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order.
+* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy, but not both.
+* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order, but not both.
 
-Examples:
-* `sort -n -asc` sorts the list of clients by name in ascending order.
+**Examples**:
+* Sort the current list of clients by name in **ascending** order.
+  * `sort -n -asc`
 
-### Exiting the program : `exit`
+    ![sort](images/sort-asc.png)
+
+* Sort the current list of clients by name in **descending** order.
+    * `sort -n -des`
 
-Exits the program.
+      ![sort](images/sort-des.png)
 
-Format: `exit`
+[Return to Table of Contents](#table-of-contents)
 
-### Locking ClientBook : `lock`
 
-Locks ClientBook.
+### Lock ClientBook with a user-selected password: `lock`
 
-Format: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
+**Purpose**: Locks ClientBook with a user-selected password.
+
+**Format**: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
 
 * Verifies the current password before locking ClientBook with the new password.
 * The `CURRENT_PASSWORD` field can be omitted if ClientBook is not yet locked.
 * When `CURRENT_PASSWORD` and `NEW_PASSWORD` fields are both omitted, ClientBook will attempt to lock itself using the last used password that is safely stored on your device.
 
-### Unlocking ClientBook : `unlock`
+:exclamation: **Note**: After setting a password, application can only be launched through Command Prompt or Terminal.
+  * Refer to [Quick Start](#quick-start) on how to launch the application through Command Prompt or Terminal.
+
+**Examples**:
+* Lock ClientBook with password `123`.
+  * `lock 123`
+
+    ![lock 123](images/lock-new-pw.png)
+
+* Change ClientBook password from `123` to `456`.
+  * `lock 123 456`
+
+    ![lock 456](images/lock-update-pw.png)    
+
+[Return to Table of Contents](#table-of-contents)
 
-Unlocks ClientBook.
 
-Format: `unlock CURRENT_PASSWORD`
+### Unlock ClientBook : `unlock`
+
+**Purpose**: Unlocks ClientBook.
+
+**Format**: `unlock CURRENT_PASSWORD`
 
 * Verifies the current password before unlocking ClientBook.
+* Password is removed after executing this command. There is no need for to enter the password to launch the application.
```

Comment:
1. **ClientBook's** password is removed.
2. Suggestion: Future launches of ClientBook will not require a password?